enricher/api/src/test/java/io/fabric8/maven/enricher/api/BaseEnricherTest.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2018 Red Hat, Inc.
+ *
+ * Red Hat licenses this file to you under the Apache License, version
+ * 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package io.fabric8.maven.enricher.api;
+
+import io.fabric8.kubernetes.api.builder.TypedVisitor;
+import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.ContainerBuilder;
+import io.fabric8.kubernetes.api.model.ContainerPortBuilder;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.EnvVarBuilder;
+import io.fabric8.kubernetes.api.model.KubernetesListBuilder;
+import java.util.HashMap;
+import java.util.List;
+import mockit.Mocked;
+import mockit.integration.junit4.JMockit;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(JMockit.class)
+public class BaseEnricherTest {
+
+    @Mocked
+    private EnricherContext enricherContext;
+
+    @Test
+    public void should_set_environment_variables_from_resources() {
```

Comment:
Hey @lordofthejars , this_underscore_format isn't consistent with the codebase. I know the the test method name is really long and will look clumsy in camelCase format but i think we should prioritize consistency of the code format here.

As per the accordance with the current format it can be simply,

```
@Test
public void setEnvironmentVarFromResources(....) { }
```

If there's a test annotation already, 'should' would look redundant in indicating assertions.

My $0.02.